src/canvas-view/canvas-editor/canvas-editor.less

```diff
@@ -39,7 +39,7 @@
       vertical-align: middle;
     }
     .show-hide-connectors-label {
-      vertical-align: text-bottom;
+      vertical-align: 2px;
```

Comment:
If I remove the setting I get:

![image](https://cloud.githubusercontent.com/assets/11633780/23081936/1ce4f2a4-f525-11e6-8710-89b5eec4bdf9.png)